src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,98 @@
-export const Pagination = () => {};
+import { useCallback } from 'react';
+
+import cn from 'classnames';
+
+type Props = {
+  total: string[],
```

Comment:
Change the type of `total` from `string[]` to `number` as it should represent the total number of items, not an array of strings.